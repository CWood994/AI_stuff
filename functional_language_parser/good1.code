program
  fun fib(n)
    int retVal;
  begin
    retVal := n;
    if 1 < n then
      retVal := call fib(n-1) + call fib(n-2);
    endif;
    return retVal;
  end;
  int x;
begin
  input x;
  while !(x = 0)
  begin
    output x;
    output call fib(x);
    input x;
  endwhile;
end