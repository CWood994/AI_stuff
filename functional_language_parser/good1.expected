5
5
10
55
15
610
25
75025
30
832040